CHANGELOG.md

```diff
@@ -7,7 +7,8 @@
   * `cortex_bucket_stores_gate_queries_concurrent_max`
   * `cortex_bucket_stores_gate_queries_in_flight`
   * `cortex_bucket_stores_gate_duration_seconds`
-* [CHANGE] Metric `cortex_ingester_flush_reasons` has been renamed to `cortex_ingester_series_flushed_total`, and is now incremented during flush, not when series is enqueued for flushing. #2802
+* [CHANGE] Metric `cortex_ingester_flush_reasons` has been renamed to `cortex_ingester_flushed_series_total`, and is now incremented during flush, not when series is enqueued for flushing. #2802, #2818
+* [CHANGE] Added `cortex_ingester_flushing_enqueued_series_total` with `reason` label (works the same as `cortex_ingester_flush_reasons` did before) and `cortex_ingester_flushing_dequeued_series_total` with `outcome` label (superset of reason). #2818
```

Comment:
Isn't `cortex_ingester_flushing_dequeued_series_total` also a superset of `cortex_ingester_flushed_series_total` now?
Do we need both?

Reply:
Not quite... `cortex_ingester_flushing_dequeued_series_total` will reflect possible flush errors, while `cortex_ingester_flushed_series_total` is incremented before flushing.

> Do we need both?
Likely not. I will remove `cortex_ingester_flushed_series_total`.